lib/challenge.js

```diff
@@ -34,6 +46,11 @@ var pet_owner;
 
 // Answer Starts Here
 
+var pet_owner = {
+  name: 'Christine',
+  address: '79 Chestnut St, Wilmington, MA 01887',
```

Comment:
This is a good object literal.

The interpreter won't complain about that final comma, but recall that the comma is a separator character for key-value pairs. Think of it like how we type spaces to separate words in a sentence. Following this analogy, the closing curly brace is like a period, and our sentence is the object literal, starting with `{` and ending with `}`. Putting a comma after the final key-value pair, then, is like putting a space between the final word and the period that ends the sentence .